deselected_tests.yaml

```diff
@@ -208,11 +208,6 @@ deselected_tests:
   # sufficient accuracy (similar to previous cases)
   - linear_model/tests/test_coordinate_descent.py::test_enet_sample_weight_consistency >=0.23
 
-  # The regression coefficients for multi-target problem differ from scikit-learn for small datasets.
-  # Coefficients matches for first label only and the coefficients are close for larger datasets.
-  # See: https://github.com/IntelPython/daal4py/issues/275
-  - linear_model/tests/test_ridge.py::test_ridge_cv_individual_penalties >=0.24
```

Comment:
I looked at logs and tbh not sure why this doesn't show up, may need to take a closer look

Reply:
public ci seems to run and pass this test btw